test/KoreBuild.FunctionalTests/KoreBuild.FunctionalTests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
```

Comment:
I'm wary of doing this - if we were to bootstrap from scratch, this ends up being a cycle. Can we just copy this file from source \ use the one that's already here?